grammars/javascript.cson

```diff
@@ -2052,7 +2053,9 @@
                   (?:
                     [\\w$\\s]* |                        # [foo=bar] Unquoted
                     "[^"]*"    |                        # [foo="bar"] Double-quoted
-                    '[^']*'                             # [foo='bar'] Single-quoted
+                    '[^']*'    |                        # [foo='bar'] Single-quoted
+                    {[^{}]*}   |                        # [foo={a:1}] Object literal
+                    \\[ [^\\]\\[]* \\]                  # [foo=[1,2]] Array literal
```

Comment:
This would make more sense if the inside was `[^\\[\\]]*` like the object one above.